discovery-infra/test_infra/config/_cluster_config.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
```

Comment:
to the best of my knowledge, we use Python 3.6 that doesn't have it
it has been changed?

Reply:
there is a backport for this  https://pypi.org/project/dataclasses/